testsuite_how_to.md

```diff
@@ -0,0 +1,199 @@
+How to Run the Test Suite and Add Tests
+========================
+This page is meant to help familiarize a new user with using the test suite. It explains how to run the example Java test suite,
+and how to add more tests to the suite. 
+
+For an overview of the test suite, refer to [Test Suite Overview](testsuite_overview.md)
+
+How to Run the Example Java Suite
+--------------------
+The test suite itself is language-independent, and requires additional language support before tests are run on a Rest.li
```

Comment:
Update accordingly based on our term "Spec" for language-independent part.  "requires additional language support" is not clear to me